src/codegen/flags.rs

```diff
@@ -35,8 +35,9 @@ pub fn generate(env: &Env, root_path: &Path, mod_rs: &mut Vec<String>) {
     let path = root_path.join("flags.rs");
     file_saver::save_to_file(path, env.config.make_backup, |w| {
         let mut imports = Imports::new(&env.library);
-        imports.add(env.main_sys_crate_name());
+        imports.add(&format!("crate::{}", env.main_sys_crate_name()));
         imports.add("glib::translate::*");
+        imports.add("bitflags::bitflags");
```

Comment:
We re-export `bitflags` from `glib` btw, so maybe use that version directly. Then all the crates don't have to depend on it separately anymore.

Reply:
I suggest to do it in another PR (maybe the one that will remove all unneeded sys dependencies?)

Reply:
Create an issue then, but that should be exactly two lines of changes in gir or not? :)

Reply:
https://github.com/gtk-rs/gir/issues/997